dpdata/formats/xyz/quip_gap_xyz.py

```
@@ -3,25 +3,68 @@
+# Possible keys for the energy field in the extxyz comment line,
+# checked in order of priority.
+_ENERGY_KEYS = ("energy", "Energy", "free_energy", "REF_energy", "energies")
+
+
+def _parse_stress_to_virials(stress_str, cell, stress_sign=-1):
+    """Convert a stress field string to virial tensor.
+
+    Parameters
+    ----------
+    stress_str : str
+        Space-separated stress values.  Accepts either 9 values (3×3 matrix,
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)